src/scripts/loadtest/hibp.js

```diff
@@ -11,33 +11,55 @@
  * @see https://grafana.com/docs/k6/latest/get-started/running-k6
  */
 
-/* eslint-disable import/no-unresolved */
-/* eslint-disable no-undef */
-
 import { post } from "k6/http";
+import crypto from "k6/crypto";
+
+/** @type {typeof process.env} */
+const envVars = __ENV;
+
+const HIBP_NOTIFY_TOKEN = envVars.HIBP_NOTIFY_TOKEN;
+if (typeof HIBP_NOTIFY_TOKEN !== "string") {
+  throw new Error(
+    "Make sure to set the following environment variable: HIBP_NOTIFY_TOKEN",
+  );
+}
 
-const url = `${__ENV.SERVER_URL}/api/v1/hibp/notify`;
+const VIRTUAL_USERS =
+  typeof envVars.K6_VIRTUAL_USERS === "string"
+    ? parseInt(envVars.K6_VIRTUAL_USERS, 10)
+    : typeof envVars.K6_VUS === "string"
+      ? parseInt(envVars.K6_VUS, 10)
+      : 1000;
```

Comment:
I arbitrarily took 1000 as a default value here, but I don't know how many requests HIBP sends in practice - that would probably be a better value.

Reply:
It really depends on the breach... I think that's a good default, especially if running against a less-provisioned environment than our production instance.